src/Moq/It.cs

```diff
@@ -234,6 +258,29 @@ public static TValue IsIn<TValue>(IEnumerable<TValue> items)
 			return Match<TValue>.Create(value => items.Contains(value), () => It.IsIn(items));
```

Comment:
@stakx 
I left it intact so that failing output is the one without the comparer.